File: backend/src/main/java/com/example/demo/dto/TransferRequest.java
package com.example.demo.dto;

/**
 * Clase TransferRequest del sistema de inventario.
 */
public class TransferRequest {

    private String originCode;
    private String destinationCode;

    public String getOriginCode() {
        return originCode;
    }

    public void setOriginCode(String originCode) {
        this.originCode = originCode;
    }

    public String getDestinationCode() {
        return destinationCode;
    }

    public void setDestinationCode(String destinationCode) {
        this.destinationCode = destinationCode;
    }
}

File: AppAndroid/TFGApp/app/src/main/java/com/example/tfgapp/data/model/TransferRequest.kt
// Archivo comentado: app/src/main/java/com/example/tfgapp/data/model/TransferRequest.kt
// Explica la responsabilidad de esta clase y los pasos principales del flujo.

package com.example.tfgapp.data.model

// Modelo de datos usado para enviar o recibir información de la API.
data class TransferRequest(
    val originCode: String,
    val destinationCode: String
)
